packages/lib/hooks/useTheme.tsx

```diff
@@ -11,6 +11,7 @@ import { useEmbedTheme } from "@calcom/embed-core/embed-iframe";
  */
 // eslint-disable-next-line @typescript-eslint/ban-types
 export default function useTheme(themeToSet: "system" | (string & {}) | undefined | null, getOnly = false) {
+  themeToSet = themeToSet ? themeToSet : "system";
```

Comment:
I don't think this is best practice, there are certain checks and conditions when themeToSet is undefined or null.

Instead of resetting it, can we handle this when calling this hook? For example, when calling this hook from the appearance component, can we not do it like this: `useTheme(user?.appTheme ?? "system")`?